### ISSUE TITLE
[Bug] Wrong order of comments in a thread

### ISSUE THREAD

**Issue by M-casado** [DataStewardshipPortal/dsp-common#286]
### What web browser is used when the issue appears?

Chrome

### How are you using DSW while encountering the issue?

- [ ] Testing on localhost (dsw-deployment-example)
- [X] Using DSW managed by someone else (specify in text)
- [ ] Using DSW that I manage

### What parts of DSW are affected?

- [X] Dashboard
- [ ] Documents
- [ ] Document Template Editors
- [ ] Document Templates
- [ ] Locales
- [ ] Knowledge Model Editors
- [ ] Knowledge Models
- [ ] Project Importers
- [ ] Projects
- [ ] Settings
- [ ] Users and Authentication

### Version

```plain text
Client
Version: v4.5.2~4c8f596
Built at: 2024-04-08T11:24:32Z
---
Server
Version: v4.5.1~aa091b89
Built at: 2024-04-09T06:40:44Z
---
Document Worker
Version: v4.5.0~42c7d6b
Built at: 2024-04-02T10:29:33Z
---
Mailer
Version: v4.5.0~42c7d6b
Built at: 2024-04-02T10:29:31Z
```

### Summary
I noticed that the comments are wrongly ordered, or directly not ordered on timestamp, which hinders following a comment thread effectively.

### Steps to reproduce
1. Click on ``Comments`` within a project
2. Leave multiple comments at different times by different people.
3. Observe how the comments are unordered, albeit the dates and times of each comment are correctly displayed.

See example bellow:
- Following the date and time of each comment, the order, from top to bottom, of the comments are: 1, 3, 4, 2
<img width="229" alt="image" src="https://github.com/ds-wizard/ds-wizard/assets/77288380/00a00862-3b9d-4dfd-becc-e2801ac1394c">

### Additional context
I tested by making comments myself, all within minutes, and by the same user, and they came out rightly ordered. Perhaps the issue comes when the comments are made by different users or across different days.
<img width="215" alt="image" src="https://github.com/ds-wizard/ds-wizard/assets/77288380/747e49ab-b847-4fcb-9620-1f19b83c5afc">

**Comment by krystofkomanec** [comment]
Hi @M-casado, thank you for the report.

I was able to reproduce it and forwarded it to our developers.

I will notify you once it has been fixed.

**Comment by krystofkomanec** [comment]
This issue has been resolved in 4.6 https://changelog.ds-wizard.org/#v4-6.
